dashboards-observability/public/components/explorer/visualizations/config_panel/config_editor/config_controls/config_thresholds.tsx

```diff
@@ -30,7 +31,7 @@ export const ConfigThresholds = ({
   let addButtonText = '+ Add threadshold';
   const getThresholdUnit = () => {
     return {
-      thid: uniqueId('thr'),
+      thid: htmlIdGenerator('thr'),
```

Comment:
Just curious: Why the change??

Reply:
this relates to an issue where i found unique id was same for previously saved threshold vs newly added threshold, in that case, changing the newly added threshold would change the saved one also since they both have same th.id